### PR TITLE
REL-1331 v24.2.1: [Docs] Generate release notes for Cloud

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes REL-1331

unedited release notes

### CODE REVIEW

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - The [`node_shutdown_transaction_timeout`]({% link v24.2/eventlog.md %}#node_shutdown_transaction_timeout) event is logged after the timeout defined by [`server.shutdown.transactions.timeout`]({% link v24.2/cluster-settings.md %}#setting-server-shutdown-query-wait) transpires, if there are still open [transactions]({% link v24.2/transactions.md %}) on those SQL connections. [#128712][#128712]
+
+<h3 id="v24-2-1-db-console-changes">DB Console changes</h3>
+
+- Corrected the series names in the legend for the [`Admission Queueing Delay p99 – Background (Elastic) CPU` graph]({% link v24.2/ui-overload-dashboard.md %}#admission-queueing-delay-p99-background-elastic-cpu) on the [Overload dashboard]({% link v24.2/ui-overload-dashboard.md %}) by removing the `KV write ' prefix. [#128891][#128891]
+- The user experience for graphs on the [Metrics dashboards]({% link v24.2/ui-overview.md %}#metrics) in the [DB Console]({% link v24.2/ui-overview.md %}) now has hover behavior that focuses on individual lines and shows values under the mouse pointer. [#128412][#128412]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:30]:
```suggestion
- Graphs on the [Metrics dashboards]({% link v24.2/ui-overview.md %}#metrics) in the [DB Console]({% link v24.2/ui-overview.md %}) now allow you to hover over individual lines to show their values. [#128412][#128412]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+<h3 id="v24-2-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a memory leak that could occur when specifying a non-existent [virtual cluster]({% link v24.2/cluster-virtualization-overview.md %}) name in the connection string. [#128108][#128108]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:34]:
```suggestion
- Fixed a memory leak that could occur when a connection string specifies a [virtual cluster]({% link v24.2/cluster-virtualization-overview.md %}) that does not exist. [#128108][#128108]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- Fixed a bug where [`CREATE INDEX IF NOT EXISTS`]({% link v24.2/create-index.md %}) would not correctly short-circuit if the given index already existed. [#128240][#128240]
+- Fixed a bug in overly eager syntax validation, which did not allow the `DESCENDING` clause for non-terminal columns of an [inverted index]({% link v24.2/inverted-indexes.md %}). Only the last column of an inverted index should be prevented from being `DESCENDING`, and this is now properly checked. [#128240][#128240]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:36]:
```suggestion
- Fixed a bug where syntax validation incorrectly prevented use of the  `DESCENDING` clause for non-terminal columns of an [inverted index]({% link v24.2/inverted-indexes.md %}). Now only the last column of an inverted index is prevented from using `DESCENDING`. [#128240][#128240]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- Fixed a bug where an [index]({% link v24.2/indexes.md %}) could store a column in the primary index if that column had a mixed-case name. [#128240][#128240]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:37]: Does this mean that it shouldn't store a column with a mixed-case name?

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+- Setting or dropping a default value on a [computed column]({% link v24.2/computed-columns.md %}) is now blocked -- even for null defaults. Previously, setting or dropping a default value on a computed column was a no-op; now there will be an error message. [#128465][#128465]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:38]:
```suggestion
- Setting or dropping a default value on a [computed column]({% link v24.2/computed-columns.md %}) is now disallowed, even for null defaults. Previously, setting or dropping a default value on a computed column was a no-op; now it is an error. [#128465][#128465]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- Fixed a bug where a hash-sharded constraint could not be created if it referred to columns that had a backslash in the name. [#128522][#128522]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:39]:
```suggestion
- Fixed a bug where a hash-sharded constraint could not be created if it referred to a column that had a backslash in the name. [#128522][#128522]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+- Fixed a bug in which the output of [`EXPLAIN (OPT, REDACT)`]({% link v24.2/explain.md %}) for various `CREATE` statements was not redacted. This bug has existed since [`EXPLAIN (REDACT)`]({% link v24.2/explain.md %}#parameters) was introduced in v23.1 and affects the following statements:
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:40]:
```suggestion
- Fixed a bug introduced in v23.1 where the output of [`EXPLAIN (OPT, REDACT)`]({% link v24.2/explain.md %}) for various `CREATE` statements was not redacted. This bug affects the following statements:
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+  - `EXPLAIN (OPT, REDACT) CREATE TABLE`
+  - `EXPLAIN (OPT, REDACT) CREATE VIEW`
+  - `EXPLAIN (OPT, REDACT) CREATE FUNCTION` [#128490][#128490]
+- Fixed a bug where legacy and [declarative schema changes]({% link v24.2/online-schema-changes.md %}#declarative-schema-changer) were allowed to be executed concurrently, which could lead to failing or hung schema change jobs. [#128825][#128825]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:44]:
```suggestion
- Fixed a bug where legacy and [declarative schema changes]({% link v24.2/online-schema-changes.md %}#declarative-schema-changer) could be executed concurrently, which could lead to failing or hung schema change jobs. [#128825][#128825]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+- Fixed a bug that caused errors like `ERROR: column 'crdb_internal_idx_expr' does not exist` when accessing a table with an [expression index]({% link v24.2/expression-indexes.md %}) where the expression evaluates to an [ENUM type]({% link v24.2/enum.md %}), for example, `CREATE INDEX ON t ((col::an_enum))`. [#129094][#129094]
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:45]:
```suggestion
- Fixed a bug that caused errors like `ERROR: column 'crdb_internal_idx_expr' does not exist` when accessing a table with an [expression index]({% link v24.2/expression-indexes.md %}) where the expression evaluates to an [ENUM type]({% link v24.2/enum.md %}) such as `CREATE INDEX ON t ((col::an_enum))`. [#129094][#129094]
```

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+[#128837]: https://github.com/cockroachdb/cockroach/pull/128837
+[#128891]: https://github.com/cockroachdb/cockroach/pull/128891
+[#129094]: https://github.com/cockroachdb/cockroach/pull/129094
+[30ea47da6]: https://github.com/cockroachdb/cockroach/commit/30ea47da6
+[4b7023084]: https://github.com/cockroachdb/cockroach/commit/4b7023084
+[522da4735]: https://github.com/cockroachdb/cockroach/commit/522da4735
+[5780226f8]: https://github.com/cockroachdb/cockroach/commit/5780226f8
+[806d0058d]: https://github.com/cockroachdb/cockroach/commit/806d0058d
+[ac8d06a16]: https://github.com/cockroachdb/cockroach/commit/ac8d06a16
+[ed9abfd9e]: https://github.com/cockroachdb/cockroach/commit/ed9abfd9e
```
Comment (mdlinville) [inline src/current/_includes/releases/v24.2/v24.2.1.md:72]:
```suggestion
```

**Review by mdlinville — APPROVED** [review]
A few small questions but going ahead and approving so you are not blocked.

File: src/current/_includes/releases/v24.2/v24.2.1.md (hunk 9 above)
Comment (florence-crl) [reply src/current/_includes/releases/v24.2/v24.2.1.md]: updated

File: src/current/_includes/releases/v24.2/v24.2.1.md (hunk 10 above)
Comment (florence-crl) [reply src/current/_includes/releases/v24.2/v24.2.1.md]: updated

**Review by florence-crl — COMMENTED** [review]
TFTR!

File: src/current/_includes/releases/v24.2/v24.2.1.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
+## v24.2.1
+
+Release Date: September 5, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+<h3 id="v24-2-1-security-updates">Security updates</h3>
+
+- The cluster setting [`server.jwt_authentication.issuers`]({% link v24.2/sso-sql.md %}#cluster-settings) is now changed to `server.jwt_authentication.issuers.configuration` to reflect the value the setting can take. The setting can now take multiple values to support various kinds of providers and their mapped JWKS URIs. This can be set to one of the following values:
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/_includes/releases/v24.2/v24.2.1.md:8]: answer:
> The old setting is still around as an alias, for backwards-compatibility.

**Pull request merged**